entity-management-utils/src/main/java/eu/europeana/entitymanagement/utils/jsonld/EntityJsonComparator.java

```diff
@@ -0,0 +1,122 @@
+package eu.europeana.entitymanagement.utils.jsonld;
+
+import java.util.Comparator;
+import java.util.HashMap;
+//import org.apache.stanbol.commons.jsonld.JsonLdCommon;
+
+import org.apache.stanbol.commons.jsonld.JsonLdCommon;
+
+import eu.europeana.entitymanagement.definitions.model.vocabulary.WebEntityConstants;
+
+
+
+/**
+ * This class is specific to annotations must be moved to (europeana) annotation
+ * repository A comparator for JSON-LD maps to ensure the order of certain key
+ * elements like '#', '@', 'a' in JSON-LD output.
+ *
+ * @author Jan Roerden
+ */
+
+public class EntityJsonComparator implements Comparator<Object> {
```

Comment:
we will not use Annotation-ld library for serilization but Jackson, please remove this class

Reply:
Removed.